README.md

```diff
@@ -1,2 +1,19 @@
 # code-review-bot
-code-review-bot
```

Comment:
Add a note about prerequisites for using these commands, such as requiring Homebrew for the installation of go-task, to clarify dependencies for users who might not have it installed already.